randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -283,16 +309,24 @@ Extra - room_id: [8]
   * L0 Hatch to Arctic Containment/Door to Geron Checkpoint
   * Extra - door_idx: (36,)
   > Door to Security Save Room
-      # TODO: varia / damage runs
-      Varia Suit and Wave Beam and Can Kill PB Geron
+      All of the following:
+          Wave Beam and Can Kill PB Geron
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Damage Runs (Advanced) and Cold Damage ≥ 100
 
 > Door to Security Save Room; Heals? False
   * Layers: default
   * L0 Hatch to Security Save Room/Door to Geron Checkpoint
   * Extra - door_idx: (64,)
   > Door to Arctic Containment
-      # TODO: varia / damage runs
-      Varia Suit and Can Kill PB Geron
+      All of the following:
+          Can Kill PB Geron
+          Any of the following:
+              Varia Suit
+              # Damage Reqs
+              Damage Runs (Advanced) and Cold Damage ≥ 180
```

Comment:
how come t he 80 damage discrepancy between the directions?

Reply:
this is due to you you tanking more hits from the zeela

Reply:
why not just kill the zeelas via shooting? does that not save health over running into them?

Reply:
ok trying it out i do get lower numbers, but this connection is a little weird to test with no geron